partcad/tests/unit/test_git_clone_retries.py

```
@@ -0,0 +1,128 @@
+
+    # The number of calls must be initial attempt plus 'max' retries
+    assert mock_clone_from.call_count == test_git_retry_config["max"] + 1
+
+
+def test_project_import_git_clone_retry_then_success():
+    fail_count = 3
+    def side_effect(*args, **kwargs):
+        side_effect.counter += 1
+        if side_effect.counter < fail_count:
+            raise fake_git_errors[0]
+        else:
```

Ruff (0.8.2)

Unnecessary `else` after `raise` statement

Remove unnecessary `else`

(RET506)